Documentation/Contributors/CodingGuide/README.md

```diff
@@ -135,6 +136,12 @@ function Model(options) {
 
 * Text files, including JavaScript files, end with a newline to minimize the noise in diffs.
 
+## Code Guidelines
+
+For syntax and style guidelines, we adhere to the common JavaScript "recommended" rules, according to [ESLint](http://eslint.org/docs/rules/). Some
+additions include enabling [`no-new`](http://eslint.org/docs/rules/no-new) and [`no-extend-native`](http://eslint.org/docs/rules/no-extend-native).
+For a complete list of what rules are enabled, look in `index.js`, `browser.js`, and `node.js` in the [`eslint-config-cesium`](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Tools/eslint-config-cesium) folder.
```

Comment:
After this sentence, mention that `eslint-config-cesium` is a separate Node module maintained in the Cesium repository and is available for use in other projects.